docs/FAQ/FAQ-General.md

```diff
@@ -168,6 +169,12 @@ BTCPay generates a different address for each invoice. The address is derived fr
 
 At this time, it is not possible to instantly convert your payments into fiat through BTCPay. This feature is planned for the future, but there is no ETA.
 
+## What if I have a problem with a paid invoice?
+
+BTCPay Server is an open-source self-hosted software stack, not a company. The community and contributors behind BTCPay Server have no control over who uses the software or how they use it.
```

Comment:
```suggestion
BTCPay Server is an open-source self-hosted software stack, not a company. The community and contributors behind BTCPay Server have no control over who uses the software or how they use it. Each merchant running the software controls their own store, that connects to their own wallet to receive funds. The BTCPay Server community does not hold or have access to any funds of a store using the BTCPay Server software, only the merchant does. 
```

Reply:
I feel like we should be more explicit... I'm open to any wording for this. I just used the example that was posted in the chat today.

Reply:
Good point, this part needs to be dumbed down, customers don't know nor care what btcpay is.

Reply:
Would this be okay ? @pavlenex @britttttk 

Tried to fix the repetition and keep it accurate to the original meaning.

```suggestion
BTCPay Server is an open-source self-hosted software stack, not a company. The community and contributors behind BTCPay Server have no control over who uses the software or how they use it.
Each merchant running the software controls their own store and the connected wallets that receive funds. The BTCPay Server community does not hold or have access to any funds of a store using the BTCPay Server software, only the merchant does.
```

Reply:
@britttttk I wouldn't wan't to highjack your authoring of this §. Do you wan't to suggest it again, or do I commit it directly ?

Reply:
>Do you wan't to suggest it again 

No worries @Zaxounette I just like the insert suggestion button for text changes, otherwise it can be confusing. The co-commit doesn't matter :) Your suggestion is much better and clear, thanks!